### PR TITLE
COR 287 env vars for frontend

### PR DESCRIPTION
**Pull request by sqwxl**
You should now create a .env file in `core/frontend` with the local IP of the device you have the server running on

```
SERVER_HOSTNAME='xxx.xxx.xxx.xxx:9000'
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,5 @@
 *.out
 actions/__tmp
 vendor/
-actions/__tmp
\ No newline at end of file
+api/tmp/
+api/models.ts*
```
Comment (internetti) [inline .gitignore:11]: could we also add the model backups here?

File: frontend/app.config.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import 'dotenv/config';
+
+export default {
+    extra: {
+        server_default_hostname: 'localhost:9000',
+        server_hostname: process.env.SERVER_HOSTNAME,
+    },
+};
```
Comment (internetti) [inline frontend/app.config.js:8]: how much more work would it be to enable dev/prod/local environments?

File: .gitignore (hunk 1 above)
Comment (nilueps) [reply .gitignore:11]: that's what the _*_ is for ;)

Comment (internetti) [reply .gitignore:11]: shouldn't those files be removed from this pr then?

**Pull request merged**